dwave/cloud/client/base.py

```diff
@@ -220,21 +221,32 @@ class Client(object):
             D-Wave Metadata API endpoint. Central for all regions, used for
             regional SAPI endpoint discovery.
 
+        leap_api_endpoint (str, optional):
+            D-Wave Leap API endpoint.
```

Comment:
We don't prefix D-Wave to Leap. Perhaps "Leap quantum cloud service" if you want to give more context

Reply:
How do you feel about [this](https://github.com/dwavesystems/dwave-cloud-client/blob/d6cec8b6812db2b8a6b73f68c444267d036025ec/dwave/cloud/config/__init__.py#L59) then? :slightly_smiling_face:

Reply:
It's always about context :-)